app/models/user_repos_service.rb

```diff
@@ -0,0 +1,8 @@
+class UserReposService
+
+	def self.find_repos(user)
+		response = Faraday.get("https://api.github.com/users/#{user}/repos?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")	
```

Comment:
@Dpalazzari yup! your client_id and client_secret will always be the same because those are unique to your application. the only difference in this request will be the `user`